WebHostLib/options.py

```diff
@@ -39,10 +38,10 @@ def render_options_page(template: str, world_name: str, is_complex: bool = False
     grouped_options = {group: {} for group in ordered_groups}
     for option_name, option in world.options_dataclass.type_hints.items():
         # Exclude settings from options pages if their visibility is disabled
-        if not is_complex and option.visibility < Visibility.simple_ui:
+        if not is_complex and option.visibility & Options.Visibility.simple_ui:
             continue
 
-        if is_complex and option.visibility < Visibility.complex_ui:
+        if is_complex and option.visibility & Options.Visibility.complex_ui:
```

Comment:
These lines seem to have caused the vast majority of options to disappear from the options pages.

Reply:
yep, missunderstood the double negative, fixing it in a moment.

Reply:
Done, hopefully makes the code clearer to any future readers too, preventing my mistake from being done again.

Reply:
Confirming the options are displayed again.